pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -120,6 +120,9 @@ type ShootSpec struct {
 	// ExposureClassName is the optional name of an exposure class to apply a control plane endpoint exposure strategy.
 	// +optional
 	ExposureClassName *string `json:"exposureClassName,omitempty" protobuf:"bytes,18,opt,name=exposureClassName"`
+	// SystemComponents contains the settings of system components in the control or data plane of the Shoot cluster.
+	// +optional
+	SystemComponents *SystemComponents `json:"systemComponents" protobuf:"bytes,19,opt,name=systemComponents"`
```

Comment:
Please adapt the `Shoot` example file in the `examples` folder with this new field

Reply:
`Shoot` example adapted with `systemComponents` section.